### DISCUSSION TITLE
$GITHUB_STEP_SUMMARY is not working on m5d.2xlarge

### DISCUSSION THREAD

**Discussion by sanjaymengani** [community/community#36797] [Other Feature Feedback, Questions, & Ideas]
I'm trying to build and deploy python application on the m5d.2xlarge. Need to show the Code Coverage Percentage on the summary screen. 
run: |
          echo "This is the lead in sentence for the list" >> $GITHUB_STEP_SUMMARY
          echo "" >> $GITHUB_STEP_SUMMARY # this is a blank line
          echo "- Lets add a bullet point" >> $GITHUB_STEP_SUMMARY
          echo "- Lets add a second bullet point" >> $GITHUB_STEP_SUMMARY
          echo "- How about a third one?" >> $GITHUB_STEP_SUMMARY

The same is working if runs-on: ubuntu-latest.
Please help me to resolve this issue.

Thanks in advance.

**Comment by ossanna16** [comment]
Thank you for asking a good question @sanjaymengani, I'm glad you found our community 🙂
